README.adoc

```diff
@@ -177,43 +182,43 @@ include::finish/src/main/java/io/openliberty/guides/data/Packages.java[]
 
 [role="code_command hotspot file=0", subs="quotes"]
 ----
-#Update the `Packages.java` class.#
+#Update the `Packages.java` class:#
 `src/main/java/io/openliberty/guides/data/Packages.java`
 ----
 
 === Sort
-In addition to the `@OrderBy` annotation it is possible to provide sorting at runtime. This is accomplished by adding a `Sort` as a parameter to your query method.
+In addition to the `@OrderBy` annotation it is possible to provide sorting at runtime. This is accomplished by adding a `Sort` object as a parameter to your query method:
 
 ```java
 @Find
 List<Package> sorted(Sort<?> sortBy);
 ```
 
-A `Sort` can be created and provided at runtime using the Sort class's static methods:
+A `Sort` can be created and provided at runtime using the `Sort` class's static methods:
 ```java
 Sort sort = Sort.desc("height");
 ```
 
 === Limit
-Jakarta Data queries can restrict the number of results returned at runtime by providing a `Limit` object to the query.
+Jakarta Data queries can restrict the number of entities returned at runtime by providing a `Limit` object to the query:
 
 ```java
 List<Package> longestWithLimit(Limit limit);
 ```
 
-Limits can be created and supplied using the Limit class's static methods:
+Limits can be created and supplied using the `Limit` class's static methods:
 ```java
 Limit limitTen = Limit.of(10);
 ```
 
-Limits can also be used to access a range of results, such as starting at result 10 and ending at result 20:
+Limits can also be used to access a range of packages, such as starting at package 10 and ending at package 20:
```

Comment:
```suggestion
Limits can also be used to access a range of entities, such as starting at the 10th entity and ending at the 20th:
```
Should probably use entity for consistency with above. I'm also thinking 10th and 20th maybe get the idea of range across better?